Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM quay.io/wikiwatershed/taudem:5.3.8
+
+MAINTAINER Azavea <systems@azavea.com>
+
+ENV RWD_VERSION 1.2.0
+
+COPY requirements.txt /tmp/
+COPY src/api/requirements.txt /tmp/src/api/
+RUN pip install --no-cache-dir -r /tmp/requirements.txt && \
+    pip install --no-cache-dir -r /tmp/src/api/requirements.txt && \
+    rm /tmp/requirements.txt && \
+    rm /tmp/src/api/requirements.txt
+
+ENV PYTHONPATH :$PYTHONPATH
+
+COPY ./rwd_drb /usr/src/api/rwd_drb
+COPY ./rwd_nhd /usr/src/api/rwd_nhd
```

Comment:
Things will be easier if we try to preserve the original directory structure as much as possible. I'm not sure if we'll be able to mount the source code inside the container for auto reloading if we do this.